README.md

```diff
@@ -87,91 +108,80 @@ echo -e "admin\n${CONCOURSE_ATC_PASSWORD}" | \
    ${FLY_CMD} -t ${DEPLOY_ENV} login -k -c "https://${DEPLOY_ENV}-concourse.cf.paas.alphagov.co.uk"
 ```
 
-## BOSH and Cloud Foundry
+### Destroy
 
-### Microbosh deployment from concourse bootstrap
+Run the `destroy-deployer` pipeline from your *Bootstrap Concourse*.
 
-These pipelines will deploy/destroy a microbosh using bosh-init. It will use the IAM profile `deployer-concourse` that was given to the Concourse instance.
```

Comment:
I agree with Alex. I am a firm believer in trying not duplicate code/config in documentation because people *will* forget to maintain it and out-of-date documentation is worse than no documentation at all.

The profile names are detailed in code. We can link to the terraform-iam repo near the top of the README, where we talk about access keys and instance profiles - but it'll only be of use to people within GDS.

I guess it depends who we're documenting for - people on our project, or people external to GDS wanting to re-use the project. I think we should optimise for the former at the moment.